### ISSUE TITLE
Retrain

### ISSUE THREAD

**Issue by Zumbalamambo** [AITTSMD/MTCNN-Tensorflow#9]
I want to train MTCNN using my own dataset.  can you please give me some hint to do that?, for example:

1. Format of the dataset, like naming the folder and image.
2. Train the network to produce the model

**Comment by AITTSMD** [comment]
@Zumbalamambo if your dataset has both face detection label and face landmark detection label,that will be great.You just need to follow my procedure in **README**. If your dataset is big,the result will also be good.

**Comment by Zumbalamambo** [comment]
@AITTSMD May I know the format of face detection label and landmark detection label?

**Comment by Zumbalamambo** [comment]
I have the tfrecord for face detection . I generated this using tensorflow's object detection api. can I use this with MTCNN?

**Comment by AITTSMD** [comment]
@Zumbalamambo I have described the format in　**README**. The format is : [path to image][cls_label][bbox_label][landmark_label].I recommend you should read **README** carefully. Thank you!

**Comment by srijan14** [comment]
@AITTSMD @Zumbalamambo  does it finetune the model on top of the base model, if I retrain on my own dataset?
